subgraphs/erc721-metadata/src/mappings/collection.ts

```diff
@@ -47,6 +50,16 @@ export function handleTransfer(event: Transfer): void {
     return;
   }
 
+  // ERC721 collection created via registry list (supportsERC721Metadata defaults to false until ERC721 transfer is indexed)
```

Comment:
condition for updating a collection entity already created to see if it supports ERC721 metadata (should not run for the default version of the subgraph without a token list)